server/api/batch_inventory.py

```diff
@@ -42,15 +42,25 @@
 BatchKey = Tuple[str, str]
 
 
-def batch_key_to_name(batch_key: BatchKey, tabulator_id_to_name: Dict[str, str]) -> str:
+def batch_key_to_name(
```

Comment:
Committing a bit out of order here. This commit takes the parsed data stored by the batch inventory CVR parsing and generates the files that we want from it. A later commit incudes the actual updated ES&S CVR parsing